ProjectLighthouse/Types/Entities/Notifications/NotificationEntity.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using LBPUnion.ProjectLighthouse.Types.Entities.Profile;
+using LBPUnion.ProjectLighthouse.Types.Notifications;
+using LBPUnion.ProjectLighthouse.Types.Serialization;
+
+namespace LBPUnion.ProjectLighthouse.Types.Entities.Notifications;
+
+public class NotificationEntity
+{
+    [Key]
+    public int Id { get; set; }
+
+    public int UserId { get; set; }
+
+    #nullable enable
+
+    [ForeignKey(nameof (UserId))]
+    public UserEntity? User { get; set; }
+
+    #nullable disable
+
+    public NotificationType Type { get; set; } = NotificationType.ModerationNotification;
+
+    public string Text { get; set; } = "";
+
+    public bool IsDismissed { get; set; } = false;
+
+    public static GameNotification ConvertToGame(NotificationEntity notification) => new()
+    {
+        Type = notification.Type,
+        Text = notification.Text,
+    };
```

Comment:
This should be in the GameNotification class and be named `CreateFromEntity` to follow the convention of other Game classes